### PR TITLE
core: added incomplete message Fix #69

### LINKED ISSUE #69: Declare Datasets and parent Containers "Partially Available" when first replica gets removed from dataset
- ~~Decide on Message format for ActiveMQ Message to AMI:~~ #70 
- ~~Schema change for database to add Partially Available(Imcomplete) to state column~~
- ~~Setting dataset/container state to PA and send message to AMI when first file gets deleted in Dataset~~ #71 

_Migrated from JIRA [RUCIO-2809](https://its.cern.ch/jira/browse/RUCIO-2809)_

### PR DESCRIPTION
**Pull request by vingar**

### CODE REVIEW

File: lib/rucio/core/replica.py [hunk 1]
```diff
@@ -1083,12 +1084,15 @@ def delete_replicas(rse, files, ignore_availability=True, session=None):
     replica_rse = get_rse(rse=rse, session=session)
 
     if not (replica_rse.availability & 1) and not ignore_availability:
-        raise exception.ResourceTemporaryUnavailable('%s is temporary unavailable for deleting' % rse)
+        raise exception.ResourceTemporaryUnavailable('%s is temporary unavailable'
+                                                     'for deleting' % rse)
```
Comment (bari12) [inline lib/rucio/core/replica.py:1088]: This looks weird, I think an additional newline went in there?

Comment (vingar) [reply lib/rucio/core/replica.py:1088]: yes it looks weird. it's one way to break string on several lines. 
>>> print ('aaaa%s'
...     'bbbb' % 'd')
aaaadbbbb

Comment (bari12) [reply lib/rucio/core/replica.py:1088]: First time I am seeing this 😄

Comment (vingar) [reply lib/rucio/core/replica.py:1088]: it was not the most intuitive way for me too.  I got some inspiration from:
https://mail.python.org/pipermail/python-list/2013-November/661383.html

**Review by bari12 — APPROVED** [review]

**Pull request merged**